### PR TITLE
Add LightStep tracing docs

### PR DESCRIPTION
**Pull request by JulianGriggs**
Instructions for running the website locally can be found [here](https://github.com/istio/istio.io). You will need to have docker installed as well.

### CODE REVIEW

File: content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+title: Distributed Tracing With LightStep
```
Comment (dchu) [inline content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md]: Distributed Tracing With LightStep [𝑥]PM

File: content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+description: How to configure the proxies to send tracing requests to LightStep.
```
Comment (dchu) [inline content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md]: How to configure the proxies to send tracing requests to LightStep [𝑥]PM.

File: content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+weight: 11
+keywords: [telemetry,tracing,lightstep]
+---
+
+This task shows you how to configure Istio to collect trace spans and send them to LightStep.
```
Comment (dchu) [inline content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md]: This task shows you how to configure Istio to collect trace spans and send them to LightStep [𝑥]PM.
[𝑥]PM lets you analyze 100% of unsampled transaction data from large-scale production software to produce meaningful distributed traces and metrics that help explain performance behaviors and accelerate root cause analysis. For more information, visit [LightStep](https://lightstep.com).
At the end of this task, trace spans will be sent from the proxies, to a LightStep [𝑥]PM Satellite pool, and then be consumable in the web UI.

File: content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+The [Bookinfo](/docs/examples/bookinfo/) sample is used as the
+example application for this task.
+
+## Before you begin
+
+> {{< idea_icon >}} The following instructions assume you have a LightStep account and a Satellite pool configured with
+TLS certs and a secure GRPC port exposed. To create an account, [contact LightStep](https://lightstep.com/contact/).
+For details about setting up Satellites, see [LightStep Satellite Setup](https://docs.lightstep.com/docs/satellite-setup).
+
+1.  Make sure that you have a LightStep Access Token and that the Satellite pool can be reached at an address of the form
```
Comment (dchu) [inline content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md]: "of the form" -> "in the form of"

File: content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+    For the
+    [Bookinfo](/docs/examples/bookinfo/#determining-the-ingress-ip-and-port)
+    sample, visit `http://$GATEWAY_URL/productpage` in your web browser
+    or issue the following command:
+
+    {{< text bash >}}
+    $ curl http://$GATEWAY_URL/productpage
+    {{< /text >}}
+
+1.  Load the LightStep web UI and find the query builder bar at the top of the Explorer. This query builder
```
Comment (dchu) [inline content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md]: Load the LightStep [𝑥]PM web UI then navigate to Explorer and find the query bar at the top. The query bar allows you to interactively filter results by a Service, Operation, and/or Tag values. Select `productpage.default` from the Service drop-down list and click "Run". You should see something similar to the following:

File: content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+categorizes query terms by Service, Operation, and Tags. Select `productpage.default` from the Service drop-down list and click
+"Run". You should see something similar to the following:
+
+{{< image width="100%" ratio="50%"
+    link="./istio-tracing-list-lightstep.png"
+    caption="Explorer"
+    >}}
+
+If you click on the top trace, you should see the details corresponding to your refresh of the `/productpage`.
```
Comment (dchu) [inline content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md]: (Does this mean the top trace in the table below the latency histogram? If so, I would just clarify something like...)

If you click on the first row in the table of example traces below the latency histogram, you should see the details corresponding to your refresh of the /productpage. The page should look something like this:

File: content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+    link="./istio-tracing-details-lightstep.png"
+    caption="Detailed Trace View"
+    >}}
+
+As you can see, the trace is comprised of a set of spans,
+where each span corresponds to a Bookinfo service invoked during the execution of a `/productpage` request.
+
+Every RPC is represented by two spans in the trace. For example, the call from `productpage` to `reviews` starts
+with the span labeled with operation `reviews.default.svc.cluster.local:9080/*` and service
+`productpage.default: proxy client`, which represents the client-side span for the call. It took 15.30ms . The second
```
Comment (dchu) [inline content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md]: extra space before period "It took 15.30ms ."

File: content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+span (labeled with operation `reviews.default.svc.cluster.local:9080/*` and service `reviews.default: proxy server`)
+is a child of the first span and represents the server-side span for the call. It took 14.60ms .
```
Comment (dchu) [inline content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md]: extra space before period "It took 14.60ms ."

File: content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+
+## Understanding what happened
+
+To develop an understanding of how this trace span data is generated and then propagated, see the discussion in [Distributed Tracing With Jaeger](../distributed-tracing/#understanding-what-happened).
+
+## Trace sampling
+
+Istio captures a trace for all requests by default but you can lower the trace sampling percentage as needed.
+For information on how to do so, see the discussion in [Distributed Tracing With Jaeger](../distributed-tracing/#trace-sampling).
+When using LightStep, reducing the trace sampling percentage is not recommended. In order to handle a high traffic mesh,
```
Comment (dchu) [inline content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md]: LightStep [𝑥]PM

File: content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+consider scaling up the size of your Satellite pool.
+
+## Cleanup
+
+If you are not planning any follow-up tasks, remove the Bookinfo sample application and any LightStep secrets
+from your cluster.
+
+1. To remove the Bookinfo application, refer to the [Bookinfo cleanup](/docs/examples/bookinfo/#cleanup) instructions.
+
+1. Remove the secret generated for LightStep
```
Comment (dchu) [inline content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md]: LightStep [𝑥]PM

File: content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+## Trace sampling
+
+Istio captures a trace for all requests by default but you can lower the trace sampling percentage as needed.
+For information on how to do so, see the discussion in [Distributed Tracing With Jaeger](../distributed-tracing/#trace-sampling).
+When using LightStep, reducing the trace sampling percentage is not recommended. In order to handle a high traffic mesh,
+consider scaling up the size of your Satellite pool.
+
+## Cleanup
+
+If you are not planning any follow-up tasks, remove the Bookinfo sample application and any LightStep secrets
```
Comment (dchu) [inline content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md]: LightStep [𝑥]PM

**Comment by JulianGriggs** [comment]
@dchu I made the updates you suggested. Can you review this one more time? If it looks good, can you "approve" it?

**Pull request closed without merging**